Throws an error on GitlabCI

It works properly on gitlab ci but at the end it throws strange error

```
events.js:292
      throw er; // Unhandled 'error' event
      ^
Error: spawn ps ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:267:19)
    at onErrorNT (internal/child_process.js:469:16)
    at processTicksAndRejections (internal/process/task_queues.js:84:21)
Emitted 'error' event on ChildProcess instance at:
    at Process.ChildProcess._handle.onexit (internal/child_process.js:273:12)
    at onErrorNT (internal/child_process.js:469:16)
    at processTicksAndRejections (internal/process/task_queues.js:84:21) {
  errno: 'ENOENT',
  code: 'ENOENT',
  syscall: 'spawn ps',
  path: 'ps',
  spawnargs: [ '-A', '-o', 'ppid,pid,stat,comm' ]
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
```

package.json
```
    "test:unit": "jest src/*",
    "server": "cross-env ENTRY_POINT=main webpack --config webpack.dev.js && http-server dist -p 3330",
    "test:e2e": "start-test server http://localhost:3330 test:unit",
[gitlab-ci.output.txt](https://github.com/bahmutov/start-server-and-test/files/5161701/gitlab-ci.output.txt)

```

--- a/README.md
+++ b/README.md
@@ -16,6 +16,12 @@ Requires [Node.js](https://nodejs.org/en/) version 22, 24 or above.
 npm install --save-dev start-server-and-test
 ```
 
+If you are running on Linux, ensure that `ps` is available.
+Minimal Docker images, such as `node:slim`, may not include this by default.
+On Debian, `ps` is included in the `procps` package, which may need to be installed additionally.
+Detailed requirements are listed in the dependency documentation
+[tree-kill > Methods](https://github.com/pkrumins/node-tree-kill#methods).
+
 ## Upgrade
 
 ### v1 to v2

